resources/inspectionDescriptions/GoStringIndexIsByte.html

```diff
@@ -0,0 +1,21 @@
+<!--
+  ~ Copyright 2013-2016 Sergey Ignatov, Alexander Zolotov, Florin Patan
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<html>
+<body>
+Compare string index to string literal instead of byte
```

Comment:
This description also shows in inspections window and it must tell what inspection does
![image](https://cloud.githubusercontent.com/assets/6865824/19071053/d399fb0c-8a36-11e6-8318-412dece710cf.png)